### PR TITLE
Fix seg fault possibility if InterpolatedImage is fully off the target image

### PR DESCRIPTION
**Pull request by rmjarvis**
Fixes seg fault bug reported in #1164.

Let me know if you need a bugfix release for this on the 2.3 series, or if this can just be included in 2.4.

### CODE REVIEW

File: tests/test_interpolatedimage.py [hunk 1]
```diff
@@ -1839,6 +1839,39 @@ def test_depixelize():
...
+            ymax=12
+        ),
+        dtype=float,
+        scale=1
+    )
+
+    image.fill(3)
+    prof.drawReal(image)
+
+    # The prolem was that the object is shifted fully off the target image and that was leading
```
Comment (leeskelvin) [inline tests/test_interpolatedimage.py:1867]: small typo (problem)

Comment (rmjarvis) [inline tests/test_interpolatedimage.py:1867]:
```suggestion
    # The problem was that the object is shifted fully off the target image and that was leading
```

Comment (rmjarvis) [reply tests/test_interpolatedimage.py]: Thanks.  Fixed.

Comment (rmjarvis) [reply tests/test_interpolatedimage.py]: Thanks.  Fixed.

**Review by jmeyers314 — APPROVED** [review]
LGTM.

**Comment by jmeyers314** [comment]
BTW - I do think this would be good for a backport to 2.3.

**Comment by rmjarvis** [comment]
Released v2.3.5 with this fix.  (Wheels are turning getting it onto pip and conda.)

**Pull request merged**